### DISCUSSION TITLE
调用合约报错 INVALID_MAIN_FUNCTION_SIGNATURE

### DISCUSSION THREAD

**Discussion by destinyyg** [NonceGeek/Move-Camp#144] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76560946/197314862-45115d6e-b19f-4c45-8e1f-1f7892779cfd.png)

**Comment by v1xingyue** [comment]
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/974169/197348099-5d13e25a-bf7e-4c0b-ba30-a2b9867e3d2f.png">
 同样的错误

**Comment by valerywai** [comment]
请问最后怎么解决的？

**Comment by zhaoge5859** [comment]
这个问题 我做的时候也碰到了，我记得好像 要不就是节点被关了，要不就是钱包地址填的不对

**Comment by zhaoge5859** [comment, marked as answer]
好像是move.toml里面的地址填的不对，你试试
<img width="891" alt="截屏2022-11-03 18 11 09" src="https://user-images.githubusercontent.com/116735858/199696024-80b9a7fc-c9d1-4d3f-b03f-d7de3320ac60.png">

**Reply by leeduckgo** [reply]
& 更新 Aptos CLI 到最新版
